EGaDS! Open Project Game/Assets/Scripts/AbilitySystem/PlayerController.cs

```diff
@@ -104,6 +113,33 @@ private bool IsSideOnWall(Vector2 side)
     private float _wallJumpTimer = -1;
     private float _currentAirJumpCount = 0;
 
+    /// <summary>
+    /// Returns a Vector2 based on the direction of the keys pressed. For dash purposes
+    /// </summary>
+    private Vector2 CheckKeyDirection()
+    {
+        // Checks which movement inputs are being pressed (indicates direction for dash)
+        var xKeys = Input.GetAxisRaw("Horizontal");
+        var yKeys = Input.GetAxisRaw("Vertical");
```

Comment:
Nit, be explicit when type isn't obvious.
```suggestion
        float xKeys = Input.GetAxisRaw("Horizontal");
        float yKeys = Input.GetAxisRaw("Vertical");
```